@shared/api/internal.ts

```diff
@@ -556,20 +556,19 @@ export const getAccountBalancesStandalone = async ({
     // eslint-disable-next-line no-plusplus
     for (let i = 0; i < Object.keys(resp.balances).length; i++) {
       const k = Object.keys(resp.balances)[i];
-      const v: any = resp.balances[k];
-      if (v.liquidity_pool_id) {
+      const v = resp.balances[k];
+      if (v.liquidityPoolId) {
         const server = stellarSdkServer(networkUrl, networkPassphrase);
         // eslint-disable-next-line no-await-in-loop
         const lp = await server
           .liquidityPools()
-          .liquidityPoolId(v.liquidity_pool_id)
+          .liquidityPoolId(v.liquidityPoolId)
           .call();
         balances[k] = {
           ...balances[k],
-          liquidityPoolId: v.liquidity_pool_id,
+          liquidityPoolId: v.liquidityPoolId,
           reserves: lp.reserves,
         };
-        delete balances[k].liquidity_pool_id;
```

Comment:
initially, we assigned `liquidity_pool_id`'s value to `liquidityPoolId` and then deleted the old `liquidity_pool_id`. I'm just simplifying the logic in `makeDisplayableBalances` so we don't need to do this anymore